src/tools/fuzzing/fuzzing.cpp

```diff
@@ -1135,7 +1142,12 @@ Expression* TranslateToFuzzReader::makeImportCallCode(Type type) {
     if ((catching && (!exportTarget || oneIn(2))) || (!catching && oneIn(4))) {
       // Most of the time make a non-nullable funcref, to avoid errors.
       auto refType = Type(HeapType::func, oneIn(10) ? Nullable : NonNullable);
-      return builder.makeCall(refTarget, {make(refType)}, type);
+      std::vector<Expression*> args = {make(refType)};
+      if (!catching) {
+        // The first bit matters here, so we can send anything.
+        args.push_back(make(Type::i32));
+      }
```

Comment:
What does this mean? Shouldn't this be either 0 or 1 depending on whether to rthrow?

Reply:
This is future-proof, as random values in the other bits might be used later, and have no downside now. I'll clarify in a comment.